src/os/ui/window/confirmtext.js

```diff
@@ -51,6 +52,10 @@ os.ui.window.ConfirmTextCtrl = function($scope, $element) {
     }
   });
 
+  $timeout(function() {
+    $element.find('.js-confirm-input').focus();
```

Comment:
Can this be married up with the `select` bit above? The intent of that is to select text in the input, and this is focusing the input. These should probably happen in the same block, and the existing `select` bit updated to use the new selector. Do these work if you call them in `$onInit`, or is that too early?

Reply:
Oh yeah, didn't even notice that was using `setTimeout`, just assumed there wasn't one since `$timeout` wasn't injected.

I can try the `$onInit` also, but I probably won't spend too much time messing with Angular stuff since this was just a nice to have that I tossed in along the way.

Reply:
Yeah no worries, I'm not even sure `$onInit` would work for these calls because I don't think the element is actually in the DOM yet.

Reply:
`$onInit` works, should be cleaned up.